packages/common-lib/src/components/FilterButton.js

```diff
@@ -35,11 +46,17 @@ export default function FilterButton({
             colorScheme='button'
             variant='outline'
             px='5'
+            _text={{
+              textTransform: 'inherit',
+              fontWeight: '500',
+              fontSize: '14px'
+            }}
             onPress={(e) => setFiltered(true)}
             rightIcon={<IconByName name='ArrowDownSLineIcon' isDisabled />}
             {..._button}
+            {..._filterButton}
           >
-            {t('FILTER')}
+            {filterButtonText ? filterButtonText : t('FILTER')}
```

Comment:
Why is the `filterButtonText` not translated here?

Reply:
we can directly send the translated data as prop like `filterButtonText={t("FILTER_BUTTON_TEXT")}`